src/Raven.Server/Web/System/AdminMonitoringHandler.cs

```diff
@@ -198,6 +198,15 @@ private DiskMetrics GetDiskMetrics()
                     result.TotalFreeSpaceInMb = totalFreeMb;
                     result.RemainingStorageSpacePercentage = percentage;
                 }
+                var diskStatsResult = Server.DiskStatsGetter.Get(ServerStore._env.Options.DriveInfoByPath?.Value.BasePath.DriveName);
```

Comment:
You need to talk with @ml054 about this. We need to add it to our monitoring plugin as well.

Reply:
Was this handled? Issue created at least?

Reply:
https://issues.hibernatingrhinos.com/issue/RavenDB-18715